Makefile.operator

```diff
@@ -0,0 +1,36 @@
+OPERATOR_DIR=../docker-images/operator
+LIB_DIR=$(OPERATOR_DIR)/lib
+SCRIPTS_DIR=$(OPERATOR_DIR)/$(PROJECT_NAME)-scripts
+RELEASE_VERSION=$(shell cat ../release.version)
+JAR=io.strimzi.$(PROJECT_NAME)-$(RELEASE_VERSION).jar
+
+docker_build: java_build
+
+all: docker_build docker_push
+clean: java_clean
+
+include ../Makefile.maven
+
+docker_build:
+	mkdir -p $(LIB_DIR)
+# Find dependent jars which are signed (contain META-INF/*.SF or META-INF/*RSA) and remove those from the jar
```

Comment:
I do not think we need this for the independent JARs. This was causing issues only in the fat jars.

Reply:
What was the issue again?

Reply:
Signatures in fat JARs.